pkg/kv/kvserver/kvflowcontrol/replica_rac2/raft_node.go

```diff
@@ -47,6 +48,9 @@ func (rn raftNodeForRACv2) FollowerStateRaftMuLocked(
 	replicaID roachpb.ReplicaID,
 ) rac2.FollowerStateInfo {
 	// TODO(pav-kv): this is a temporary implementation.
+	rn.MuRLock()
```

Comment:
For consistency, it would be better to move this lock to the caller. Eventually, it will move all the way up to `RaftEvent` construction time.

Also, how about renaming `FollowerStateRaftMuLocked` to `FollowerStateLocked`? It doesn't matter whether `raftMu` is locked, for the purpose of extracting state from `raft.RawNode`.